arch/arm/src/armv7-a/arm_cpupause.c

```diff
@@ -168,10 +212,8 @@ int up_cpu_paused(int cpu)
    */
 
   arm_restorestate(tcb->xcp.regs);
-  spin_unlock(&g_cpu_wait[cpu]);
-  spin_unlock(&g_cpu_resumed[cpu]);
 
-  return OK;
+  return 0;
```

Comment:
Please use OK instead 0. As you wrote in the Retuned Value: "On success, OK is returned." :-)

Reply:
done